src/@next/components/atoms/CartCostsSummary/test.tsx

```diff
@@ -0,0 +1,33 @@
+import { mount, shallow } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { CartCostsSummary } from ".";
+import { BASIC_COSTS, ALL_POSSIBLE_COSTS } from "./fixtures";
+
+describe("<CartCostsSummary />", () => {
+  it("exists", () => {
+    const wrapper = shallow(<CartCostsSummary {...BASIC_COSTS} />);
+
+    expect(wrapper.exists()).toEqual(true);
+  });
+
+  it("should display all costs", () => {
+    const wrapper = mount(<CartCostsSummary {...ALL_POSSIBLE_COSTS} />);
+
+    const cartSummaryCosts = wrapper.text();
+
+    expect(cartSummaryCosts).toContain(
+      ALL_POSSIBLE_COSTS.subtotalPrice?.gross.amount
```

Comment:
I'm not a fan of passing forced Locale with env var. Can we write this test like we compare prices in e2e tests? 
It is:
- take price as string
- remove currency
- parse rest as float and compare amounts?

Reply:
~~It is possible, but we have to take into account that e.g.:~~
- ~~if fixture has `10.50` and text will be `10,50`, we need first to do `replace(/,/g, '.')`~~
- ~~although if fixture has `2299.50` and text will be `2,299.50`, we need first to do `replace(/,/g, '')`~~

~~I have made `replace(/,/g, '.')` with small price as fixture, so I guess parsing error won't occur. Please review it and decide if it is ok.~~

Unfortunately, it breaks snapshots unpredictably:
![Zrzut ekranu 2020-10-2 o 15 02 26](https://user-images.githubusercontent.com/9825562/94926033-73ebd780-04c0-11eb-9cff-eea0e82e54db.png)
Problem didn't occur previously, as `Money` component had not test snapshot.

Reply:
Ok, lets roll with it